### DISCUSSION TITLE
My request is approved still didn't get the benefits yet whereas my friend got same & we both applied same time

### DISCUSSION THREAD

**Discussion by mittalkartikey11** [community/community#147383] [GitHub Education]
### Hi everyone,

My request is approved still didn't get the benefits yet whereas my friend got same & we both applied same time. Kindly do needful as I urgently needed for my semester

**Comment by cedricangulo** [comment]
Hi, it seems that there might be a delay in processing your benefits. Please wait a few more days for the benefits to reflect in your account. I experienced a similar wait when I applied for the GitHub Student Developer Pack.

**Reply by mittalkartikey11** [reply]
oh okay..will wait more then

Thanks

**Comment by davevad93** [comment]
Hi @mittalkartikey11 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @mittalkartikey11 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
